### ISSUE TITLE
One candidate with inconsistent totals 

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/openFEC#3925]
After https://github.com/fecgov/openFEC/pull/3919 is merged, there is an outstanding issue with one candidate's totals.

One candidate whose candidate profile page numbers don’t match the candidates/totals/ and /elections/ totals:

```
**** Checking: H6OH04082, JORDAN, JAMES D. in local. Election year is 2020. ****


Candidate totals datatable: http://localhost:5000/v1/candidates/totals/?candidate_id=H6OH04082&election_year=2020&election_full=true&sort=-receipts&api_key=DEMO_KEY

Candidate profile totals: http://localhost:5000/v1/candidate/H6OH04082/totals/?cycle=2020&api_key=DEMO_KEY&election_full=False

Election profile totals: http://localhost:5000/v1/elections/?candidate_id=H6OH04082&cycle=2020&election_full=true&office=house&sort_nulls_last=true&sort=-total_receipts&per_page=100&api_key=DEMO_KEY&state=OH&district=04&election_full=False

!!! ERROR - receipts results don't match!!!

| Data source | Total receipts |
|--|--|
| datatable datatable or profile page         |    $1,223,188.44|
| candidate datatable or profile page         |    $1,239,638.82|
| election datatable or profile page         |    $1,223,188.44|

!!! ERROR - disbursements results don't match!!!

| Data source | Total disbursements |
|--|--|
| datatable datatable or profile page         |    $873,828.77|
| candidate datatable or profile page         |    $874,928.77|
| election datatable or profile page         |    $873,828.77|

!!! ERROR - cash_on_hand_end_period results don't match!!!

| Data source | Total cash_on_hand_end_period |
|--|--|
| datatable datatable or profile page         |    $1,072,035.02|
| candidate datatable or profile page         |    $1,087,385.40|
| election datatable or profile page         |    $1,072,035.02|
```

From @fecjjeng:
This candidate has two committees that has cmte_dsgn in ('P', 'A').
cmte_id; cmte_tp, cmte_dsgn
"C00693796";"N";"A"
"C00416594";"H";"P"

"C00693796" files F3X and "C00693796" files F3.  `ofec_candidate_totals_mv` takes values from only F3 and F3P, while the new `ofec_candidate_totals_detail_mv` takes values from all F3/F3P/F3X, since sometimes committees file the incorrect form. The difference ($16450) in the new `ofec_candidate_totals_detail_mv` is from "C00693796"'s MY report. (edited)

From @fec-jli:
Do we need create a ticket to adjust `ofec_candidate_totals_mv`? match with new `ofec_candidate_totals_details_mv`? 

From @fecjjeng:
So the questions has two points:
1. Are we going to include committees that are NOT cmte_tp H/P/S with cmte_dsgn in ('P', 'A') to be included?  There are 154 rows in the linkage tables, which are always included in the past.
2. Are we going to include F3X data in the calculation?

**Comment by fecjjeng** [comment]
I had updated ofec_candidate_totals_mv and ofec_totals_house_senate_mv to include all F3/F3P/F3X and exclude non_candidate P/A committees.
PR https://github.com/fecgov/openFEC/pull/3935 had been created for this work.

**Comment by fecjjeng** [comment]
Paul had update data had correct the cmte_dsgn for C00693796. It should be a leadership PAC (cmte_dsgn='D'), not 'A'. This would cure the symptom of this particular candidate.  But further work in PR #3932 (for ticket #3907, already merged) and #3935 are needed to have a more complete fix of situation like this.

**Comment by fecjjeng** [comment]
PR approved and merged.  Close ticket.
